src/openrct2/ride/Vehicle.h

```diff
@@ -21,12 +21,10 @@
 #include "../world/map.h"
 
 #pragma pack(push, 1)
-/* size: 0x2 */
```

Comment:
are these not stored in sv6 or other files?

Reply:
They are, but all are loading is now done by reading each field individually.